brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3032: Fix - Image - Setting up the hover image inside the container, chopping the image part

### Description

Image - Setting up the hover image inside the container, chopping the image part
Image will get chopped on the few sides

### Screenshots

https://d.pr/v/1YYrUt

### Types of changes





### How has this been tested?


- Create a new page.
- Add Container of two columns.
- Add Heading on the left.
- Add Image on the right.
- Now setup the image “On Hover Image” as “Zoom In”.
- Check both backend and front-end.

### Checklist:
